### ISSUE TITLE
Question about imagenet_fast vanilla performance

### ISSUE THREAD

**Issue by Neonsign247** [snu-mllab/PuzzleMix#10]
Hello. Thank you for your great work.

Currently, I'm trying to reproduce vanilla performance with your imagenet_fast protocol. However, my model 2-3% behind with your configuration. Can you share tricks (data augmentation or optimized configuration) or training log?

**Comment by Janghyun1230** [comment]
Hello, 

I used the same training configuration (learning rate, augmentation) for Vanilla and Puzzle Mix, except for mixup.
 (reference code: https://github.com/anonymous-sushi-armadillo/fast_is_better_than_free_imagenet )
Currently, I don't have log files for Vanilla models. 

I guess it could be a problem caused by different versions of apex. 
Could you reproduce the results with Puzzle Mix?

If the problem maintains, I recommend to use more stable ImageNet training codes without apex:
 https://github.com/snu-mllab/PuzzleMix/tree/master/imagenet
